backend/infrahub/core/schema/generated/attribute_schema.py

```diff
@@ -113,7 +117,7 @@ class GeneratedAttributeSchema(HashableModel):
         description="Type of allowed override for the attribute.",
         json_schema_extra={"update": "allowed"},
     )
-    parameters: AttributeParameters = Field(
+    parameters: AttributeParameters | TextAttributeParameters | NumberPoolParameters = Field(
```

Comment:
Why do we need this as they inherit from `AttributeParameters`?

Reply:
It's to have them included in the json schema for the REST api. Otherwise we'll only have `AttributeParameters` in the API schema and that doesn't contain any info so we need a union of the available types. In your PR we'll have to add the number parameter here too (or in the source this is a generated file)